src/frontend/package.json

```diff
@@ -20,13 +20,13 @@
     "lint-staged": "^8.2.1",
     "lodash": "^4.17.15",
     "moment": "^2.24.0",
-    "node-sass": "^4.13.0",
+    "node-sass": "^4.13.1",
     "prettier": "^1.19.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-redux": "^7.1.3",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.2.0",
+    "react-scripts": "^3.3.0",
```

Comment:
While these probably need to be updated, I would prefer if they are done in a separate PR. You can use 

```
git reset HEAD~1 -- src/frontend/package.json
git reset HEAD~1 -- src/frontend/package-lock.json
```

and commit that.